### PR TITLE
Upgrade dependencies

### LINKED ISSUE #845: Upgrade dependencies

### PR DESCRIPTION
**Pull request by mialbu**
Closes #845

### CODE REVIEW

File: core/src/main/java/io/neow3j/protocol/http/HttpService.java [hunk 1]
```diff
@@ -255,15 +255,14 @@ private static OkHttpClient createOkHttpClient() {
...
+            logging.level(HttpLoggingInterceptor.Level.BODY);
             builder.addInterceptor(logging);
         }
     }
 
     @Override
     protected InputStream performIO(String request) throws IOException {
-
-        RequestBody requestBody = RequestBody.create(JSON_MEDIA_TYPE, request);
+        RequestBody requestBody = RequestBody.create(request, JSON_MEDIA_TYPE);
```
Comment (gsmachado) [inline core/src/main/java/io/neow3j/protocol/http/HttpService.java:265]: Seriously? They changed the order of params keeping the same method name?! 🤯

**Review by gsmachado — DISMISSED** [review]
It looks fine to me... but still don't believe they changed the order of the parameters and kept the same method name. This is almost a crime. It's not an actual crime because this change was done in a major version upgrade... 👍

File: core/src/main/java/io/neow3j/protocol/http/HttpService.java (hunk 1 above)
Comment (mialbu) [reply core/src/main/java/io/neow3j/protocol/http/HttpService.java:265]: Yes, that's actually the case :laughing:

**Review by gsmachado — APPROVED** [review]
Alright... **Assuming** the integration tests passed (locally or remotely), as well as the unit tests, then it's good to go.

**Pull request merged**